opentracing-jaxrs2-itest/common/src/main/java/io/opentracing/contrib/jaxrs2/itest/common/rest/InstrumentedRestApplication.java

```diff
@@ -27,7 +27,7 @@ public InstrumentedRestApplication(@Context ServletContext context) {
                 AbstractJettyTest.CLIENT_BUILDER_ATTRIBUTE);
 
         if (serverTracingBuilder == null || clientTracingBuilder == null) {
-            throw new IllegalArgumentException("");
+            throw new IllegalArgumentException();
```

Comment:
You were so close to give this exception a meaningful purpose in life... :) 
`String.format("Invalid Server Tracing Builder and/or Client Tracing Builder: %s and %s", serverTracingBuilder, clientTracingBuilder)` . If you don't care about a message, then just let a NPE happen...